shop/src/main/java/ro/msg/learning/shop/entity/Location.java

```diff
@@ -0,0 +1,20 @@
+package ro.msg.learning.shop.entity;
+
+import lombok.*;
+
+import javax.persistence.Entity;
+
+@Data
+@Entity
+@EqualsAndHashCode(callSuper = true)
+@NoArgsConstructor(access = AccessLevel.PROTECTED, force = true)
+@AllArgsConstructor
+public class Location extends BaseEntity{
+
+    private String name;
+    private String addressCountry;
+    private String addressCity;
+    private String addressCounty;
+    private String addressStreetAddress;
```

Comment:
You can create a separate Entity named Address with these fields. In this way you remove duplicates because the fields are used also in Order entity.

Reply:
Fixed